src/test/__mocks__/mockFormDefinition.json

```diff
@@ -9,41 +9,46 @@
         {
           "type": "STRING",
           "required": true,
-          "warnIfEmpty": true,
```

Comment:
What is `warnIfEmpty`? Why is it being removed?

Reply:
`warnIfEmpty` means that the user will need to confirm a warning if they try to submit the form without entering a value for this question. Its like a "soft" required-ness. Also, if you re-open a submitted assessment that has empty warnIfEmpty fields, they show up with a yellow background.

I think these were added here so we can show them in storybook. I changed the behavior so Storybook story just modifies the items to set warnIfEmpty to true for all of them, for that specific story. That lets us see at a glance how all the input components handle this state. The diff for that is in this PR too. To see it in storybook, run `yarn storybook` and look at the DynamicForm stories

Reply:
Oh wonderful. Thank you for explaining!